src/main/java/com/locallampoon/fiveh/ui/ArtPanel.java

```diff
@@ -13,8 +14,9 @@ public ArtPanel() {
         pane.setEnabled(false);
         pane.setBounds(600, 0, 400, 250);
         pane.setBackground(Color.BLACK);
+        pane.setBorder(new MatteBorder(1, 1, 0, 1, Color.WHITE));
         textArea = new JTextArea();
-        textArea.setBounds(100, 0, 250, 300);
+        textArea.setBounds(100, 20, 250, 300);
```

Comment:
might be outside the scope of this ticket, but you can use the PanelStyles instead of hard-coded values. i also realize the hard-coded values were already existent and that PanelStyles wasn't created until after the hard-coded values got merged.

Reply:
Did it slightly different using an inner class, let me know if you'd like to adopt the style of the rest of the component.